packages/fluentui/react-bindings/test/styles/resolveStyles-test.ts

```diff
@@ -117,7 +117,7 @@ describe('resolveStyles', () => {
   });
 
   test('caches resolved styles for no props', () => {
-    spyOn(testComponentStyles, 'root').and.callThrough();
+    jest.spyOn(testComponentStyles, 'root');
```

Comment:
looks like these need either `mockRestore` or `mockClear` at the end of each test?